portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,89 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <div id="wrapper">
+            <div id="content">
+                <div id="header">
+                    <h1 id="intro">Hi. I'm Zian.</h1>
+                    <div id="external-links">
+                        <a href="mailto:zianchen774@gmail.com" target="_blank">
+                            <img class="icon" src="./images/mail_icon.png" 
+                                alt="mail_icon"/>
+                        </a>
+                        <a href="http://github.com/zianc">
+                            <img class="icon" src="./images/github_icon.png" 
+                                alt="github_icon"/>
+                        </a>
+                        <a href="http://linkedin.com/in/zianchen">
+                            <img class="icon" src="./images/linkedin_icon.png" 
+                                alt="linkedin_icon"/>
+                        </a>
+                    </div>
+                </div>
+                <div id="path">
+                    <div class="path-stop">
+                        <img id="path-stop-img-me" src="./images/profile_picture.jpg" 
+                            alt="profile_picture"/>
+                        <p class="path-stop-desc">Thanks for dropping by! My name is Zian Chen, a soon-to-be third year student at UCLA, working to get a Bachelor's in Computer Science. My path to where I am now has not been easy, but along the way I've picked up multiple different skills and, arguably more importantly, how to deal with failure, learn the value of criticism, and rely not only on myself but those around me. Keep scrolling to track a few stops along the way! </p>
+                    </div>
+                    <img class="path-connecter" src="./images/white_path.svg"/>
+                    <div class="path-stop">
+                        <p class="path-stop-desc">SWE Dev was the first organization 
```

Comment:
The `</p>` is alone on its line so, for symmetry it's better the `<p>` to do the same.